front/pages/api/w/[wId]/apps/[aId]/runs/index.ts

```diff
@@ -32,19 +32,9 @@ async function handler(
   auth: Authenticator,
   session: SessionWithUser
 ) {
-  let owner = auth.workspace();
-  if (!owner) {
-    return apiError(req, res, {
-      status_code: 404,
-      api_error: {
-        type: "workspace_not_found",
-        message: "The workspace was not found.",
-      },
-    });
-  }
+  let owner = auth.getNonNullableWorkspace();
```

Comment:
also, could you check that product behaviour never depends on api return (i.e. different behaviour for 400s and 500s) for this call and all the similar replacements below?

Reply:
No this can't throw because the wrapper `allowUserOutsideCurrentWorkspace` checks for the non-nullity of owner and return a proper error otherwise :+1:

Reply:
Ok thanks ! LGTM then :+1: